src/components/catalog-sort.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState, useEffect} from 'react';
+import { CamerasType } from '../types/cameras';
+
+type productTabsProps = {
+  setDiplayedCameras: (isActive: []) => void;
+  diplayedCameras: CamerasType;
+}
+
+function CatalogSort({diplayedCameras, setDiplayedCameras}: productTabsProps): JSX.Element {
+
+  const [sortType, setSortType] = useState('price');
+
+  const [sortOrder, setSortOrder] = useState('minToMax');
```

Comment:
```suggestion
  const [sortOrder, setSortOrder] = useState<'minToMax' | 'maxToMin'>('minToMax');
```
без такой типизаци будет любую строку принимать